backends/arm/operators/op_add.py

```diff
@@ -16,7 +16,7 @@
     build_rescale_from_int32,
     build_rescale_to_int32,
 )
-from executorch.backends.arm.tosa_utils import broadcast_shapes, getNodeArgs
+from executorch.backends.arm.tosa_utils import broadcast_shapes, getNodeArgs, tosa_shape
```

Comment:
Are you planning to add more ops? For instance Permute op argument needs to be updated based on what format input it gets when you update the previous node's output format.

Reply:
Yes, there are a few open PR:s with new ops. The ambition is that we can just update shapes such as permute's argument with `tosa_shape(shape, dim_order)`.